Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM nixos/nix
+
+RUN nix-channel --add https://nixos.org/channels/nixpkgs-unstable \
+    && nix-channel --update \
+    && nix-env -iA nixpkgs.nixFlakes \
+    && echo 'experimental-features = nix-command flakes' | tee -a /etc/nix/nix.conf
```

Comment:
Above run command should be replaced with:

```
RUN echo 'experimental-features = nix-command flakes' | tee -a /etc/nix/nix.conf
```

This is because `nixos/nix` image already has latest `nixpkgs-unstable` channel inside the image and latest nix version is already installed (currently that is 2.8.0). But I would also enable flakes, since I would use them later on.

Reply:
Done